zerver/openapi/zulip.yaml

```diff
@@ -14791,10 +14802,19 @@ paths:
                           password combination.
 
                           [ldap-auth]: https://zulip.readthedocs.io/en/latest/production/authentication-methods.html#ldap-including-active-directory
-                      realm_uri:
+                      realm_url:
                         type: string
                         description: |
                           The organization's canonical URL.
+                          **Changes**: New in Zulip 8.0 (feature level 192). The name `realm_url`
+                          is preferred use than `realm_uri` as later is deprecated.
+                      realm_uri:
+                        type: string
+                        deprecated: true
+                        description: |
+                          The organization's canonical URL. Alias of `realm_url`.
+                          **Changes**: Deprecated in Zulip 8.0 (feature level 192). The name
```

Comment:
feature levels in this file need to be in sync with elsewhere

Reply:
Just did! A bit messy when tracking all the level numbers..